examples/js/renderers/CSS3DRenderer.js

```diff
@@ -55,6 +56,7 @@ THREE.CSS3DRenderer = function () {
 
 	var domElement = document.createElement( 'div' );
 	domElement.style.overflow = 'hidden';
+	domElement.style.pointerEvents = 'none';
```

Comment:
CSS pointer-events is an inherits-property. this will affects all descendants such as camera element, CSS3Object elements.